include/lsst/afw/image/DistortedTanWcs.h

```diff
@@ -48,25 +48,27 @@ namespace image {
  */
 class DistortedTanWcs : public TanWcs {
 public:
+    using Transform = geom::Transform<geom::Point2Endpoint, geom::Point2Endpoint>;
+
```

Comment:
Be wary of typedefs that shadow templates.

Reply:
Removed in favor of the new typedef `geom::TransformPoint2ToPoint2`

Reply:
Sounds good.  I'd prefer something even shorter, but if that's the name we've been using in Python I can at least accept that changing it is out of scope for this ticket.

Reply:
Yes. Trying to come up with even shorter names is more than I want to try to do on this ticket. I'm in favor of shorter names if they are sufficiently descriptive, but so far have not come up with anything.